Rakefile

```diff
@@ -140,6 +141,19 @@ task :release, [:type] do |task, args|
   release.publish
 end
 
+desc "Build files with scope information for integration tests"
+
+# This command can be used to generate files with information
+# of each scope of an inspected system automatically.
+# This is usefull for the integration tests.
+
+task :inspector_files, [:ip_adress, :distribution] do |task, args|
```

Comment:
What does `distribution` mean and what are the allowed values for that? This should be made clearer.